action.yml

```diff
@@ -99,21 +74,12 @@ runs:
     - name: Atmos Apply
       if: env.ACTION == 'remediate'
       id: atmos-apply
-      uses: cloudposse/github-action-atmos-terraform-apply@v0
+      uses: cloudposse/github-action-atmos-terraform-apply@incapsulate-configs
```

Comment:
should this be versioned?

Reply:
Yes. Waiting when https://github.com/cloudposse/github-action-atmos-terraform-apply/pull/29 will be approved

Reply:
```suggestion
      uses: cloudposse/github-action-atmos-terraform-apply@v2
```